security/DOXM-MFGCERT.txt

```diff
@@ -2,46 +2,62 @@
 @startuml
 autonumber
 title OIC Device Owner Establishment Sequence \n"Manufacturer Cert" Device Owner Transfer Method
-participant "On-boarding Tool\n(UUID B0B0xxxx-...)" as OT
+participant "Onboarding Tool\n(UUID B0B0xxxx-...)" as OT
 participant "New Device\n(UUID A21Cxxxx-...)" as ND
 
 ==Device Owner Transfer Method==
 note over OT, ND
-Find new devices that are unowned and choose the Manufacturer Cert owner transfer method
+Find new devices that are unowned and choose the Manufacturer Cert owner transfer method.
 end note
 OT->ND: GET /oic/sec/doxm?owned="FALSE"
 ND->OT: RSP [{"oxms":["0","2"], "owned":"FALSE", "deviceuuid":{"A21CXXXX-..."},...}]
 OT->ND: POST /oic/sec/doxm [{..., "oxmsel":"2",...}]
 ND->OT: RSP 2.04
-OT->ND: GET /oic/sec/pstat
-ND->OT: RSP [{"isop":"FALSE", "cm":"bx0011,1110", "tm":"bx0011,1110", "om":"bx0000,0000", "sm":"bx0000,0110"}]
-note over OT, ND
-On-boarding tool tells new device how provisioning will be achieved.
-end note
-OT->ND: POST /oic/sec/pstat [{..., "om":"bx0000,0100",...}]
-ND->OT: RSP 2.04
 
 ==Perform oic.sec.oxm.mfgcert method==
+
 note over OT, ND
 The Manufacturer cert private key is used to sign handshake messages. Certificate attests device to OBT.
 end note
 note over OT, ND
 If device requires authentication of the OBT, it will resolve the OBT certificate to its embedded trust anchor. Otherwise, it will implicitly trust the OBT.
 end note
 group DTLS
-  OT->ND: ClientHello(TLS_ECDHE_ECDSA_WITH_AES_128_CCM_8)
+  OT->ND: ClientHello(TLS_ECDHE_ECDSA_WITH_AES_128_CCM_8_SHA256)
   ND->OT: HelloVerifyRequest(cookie)
-  OT->ND: ClientHello(TLS_ECDHE_ECDSA_WITH_AES_128_CCM_8,cookie)
-  ND->OT: ServerHello(TLS_ECDHE_ECDSA_WITH_AES_128_CCM_8)\nCertificate*\nServerKeyExchange(ECDH PublicKey + ECC Curve Param)\nServerHelloDone()
+  OT->ND: ClientHello(TLS_ECDHE_ECDSA_WITH_AES_128_CCM_8_SHA256,cookie)
+  ND->OT: ServerHello(TLS_ECDHE_ECDSA_WITH_AES_128_CCM_8_SHA256)\nCertificate*\nServerKeyExchange(ECDH PublicKey + ECC Curve Param)\nServerHelloDone()
```

Comment:
RFC 7251 doesn't define TLS_ECDHE_ECDSA_WITH_AES_128_CCM_8_SHA256 - I think because there isn't a TLS_ECDHE_ECDSA_WITH_AES_128_CCM_8_SHA384 and there's no reason to differentiate. So I think IETF just left it off and the original text is correct.